benchmark/compare.js

```diff
@@ -136,8 +141,10 @@ function compare() {
     if (show === 'green' && !g || show === 'red' && !r)
       return;
 
-    var r0 = util.format('%s%s: %d%s', g, nodes[0], n0.toPrecision(5), g ? reset : '');
-    var r1 = util.format('%s%s: %d%s', r, nodes[1], n1.toPrecision(5), r ? reset : '');
+    var r0 = util.format('%s%s: %d%s',
```

Comment:
Argument style consistency issue here too. I think all arguments should each be on a new line and indented like with the `spawn()` change earlier in this file. Also this first argument should go on the next line.